pandas/core/common.py

```diff
@@ -490,3 +490,13 @@ def f(x):
         f = mapper
 
     return f
+
+
+def ensure_python_int(value: Union[int, Any]) -> int:
```

Comment:
put in dtypes.cast but i don’t think u need this as we already have ensure_platform_int

Reply:
``ensure_platform_int`` returns a numpy int(32|64), so won't work here:

```python
>>> rng = range(1, 1_000_000)
>>> 999_999 in rng
133 ns ± 2.46 ns per loop
>>> np.int64(999_999) in rng
306 ms ± 66.3 ms per loop
```

Reply:
ok in any event move it to collocate with the ensure _ routines